### PR TITLE
[GSW-EARN] Fix: Fix loading, earn domain, login wallet

### PR DESCRIPTION
**Pull request by khiemldk**

### CODE REVIEW

File: packages/web/src/components/earn/earn-my-positions-no-liquidity/EarnMyPositionNoLiquidity.tsx [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-interface EarnMyPositionNoLiquidityProps { }
+interface EarnMyPositionNoLiquidityProps {
+  account: AccountModel | null;
+}
 
 const EarnMyPositionNoLiquidity: React.FC<
   EarnMyPositionNoLiquidityProps
-> = () => {
+> = ({ account }) => {
+  const balanceText = useMemo(() => `$${(Number((account?.balances[0].amount || 0)) / 1000000).toLocaleString(undefined, { maximumFractionDigits: 5 })}` || "$0", [account?.balances]);
```
Comment (jinoosss) [inline packages/web/src/components/earn/earn-my-positions-no-liquidity/EarnMyPositionNoLiquidity.tsx:15]: Use `makeDisplayTokenAmount` function

File: packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.tsx [hunk 2]
```diff
@@ -34,19 +32,21 @@ const WalletBalanceDetailInfo: React.FC<WalletBalanceDetailInfoProps> = ({
...
+        </PulseSkeleton>
+
+        {/* {loading ? (
+          <div className="value">
+            <span css={pulseSkeletonStyle({ h: 20, w: "120px" })} />
+          </div>
         ) : (
-          <span className="value">$0</span>
-        )}
+          <span className="value">
```
Comment (jinoosss) [inline packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.tsx:46]: Remove unused code.

**Review by jinoosss — COMMENTED** [review]
I left a few comments.

**Review by dongwon8247 — APPROVED** [review]

File: packages/web/src/common/values/token-constant.ts [hunk 3]
```diff
@@ -15,9 +15,9 @@ export const GNOT_TOKEN: TokenModel = {
 };
 
 export const GNOT_SYMBOL = "GNOT";
-export const GNS_SYMBOL = "GNOT";
-export const BAR_SYMBOL = "GNOT";
-export const BAZ_SYMBOL = "GNOT";
+export const GNS_SYMBOL = "GNS";
+export const BAR_SYMBOL = "BAR";
+export const BAZ_SYMBOL = "BAZ";
```
[the remark this answers is not in the document]
Comment (khiemldk) [reply packages/web/src/common/values/token-constant.ts]: done

File: packages/web/src/components/earn/earn-my-positions-no-liquidity/EarnMyPositionNoLiquidity.tsx (hunk 1 above)
Comment (khiemldk) [reply packages/web/src/components/earn/earn-my-positions-no-liquidity/EarnMyPositionNoLiquidity.tsx]: done

File: packages/web/src/components/common/my-position-card/MyPositionCard.tsx [hunk 4]
```diff
@@ -114,31 +116,30 @@ const MyPositionCard: React.FC<MyPositionCardProps> = ({
...
-    return `1 ${tokenA.symbol} = ${tokenBPriceStr} ${tokenB.symbol}`;
+    const tokenBPriceStr = numberToFormat(tokenBPrice, 6);
+    return `${tokenBPriceStr}`;
   }, [tokenB.path, tokenB.symbol, tokenPrices, tokenA.path, tokenA.symbol]);
 
-  const handleClickShowRange = (e: React.MouseEvent<HTMLDivElement>) => {
+  // const handleClickShowRange = (e: React.MouseEvent<HTMLDivElement>) => {
+  //   e.stopPropagation();
+  //   setIsHiddenStart(!isHiddenStart);
+  // };
```
[the remark this answers is not in the document]
Comment (khiemldk) [reply packages/web/src/components/common/my-position-card/MyPositionCard.tsx:136]: done

File: packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.tsx (hunk 2 above)
Comment (khiemldk) [reply packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.tsx]: done

**Comment by khiemldk** [comment]
Thanks for your comment. i have fixed them

**Pull request merged**